### DISCUSSION TITLE
What could be wrong? cant get it working...

### DISCUSSION THREAD

**Discussion by x0rkinko** [Koenkk/zigbee2mqtt#17497] [General]
[22:42:23] INFO: Preparing to start...
[22:42:23] INFO: Socat not enabled
[22:42:25] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:info  2023-04-29 22:42:31: Logging to console and directory: '/config/zigbee2mqtt/log/2023-04-29.22-42-31' filename: log.txt
Zigbee2MQTT:info  2023-04-29 22:42:31: Starting Zigbee2MQTT version 1.30.3 (commit #unknown)
Zigbee2MQTT:info  2023-04-29 22:42:31: Starting zigbee-herdsman (0.14.103)
Error: Reset error: Error: {"sequence":-1} after 10000ms
    at /app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/uart.ts:298:23
    at Queue.executeNext (/app/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32)

**Comment by simplemice** [comment]
have same problem, clean install

[19:17:11] INFO: Preparing to start...
[19:17:11] INFO: Socat not enabled
[19:17:11] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:info  2023-05-16 19:17:13: Logging to console and directory: '/config/zigbee2mqtt/log/2023-05-16.19-17-13' filename: log.txt
Zigbee2MQTT:info  2023-05-16 19:17:13: Starting Zigbee2MQTT version 1.30.4 (commit #unknown)
Zigbee2MQTT:info  2023-05-16 19:17:13: Starting zigbee-herdsman (0.14.111)
Error: Reset error: Error: {"sequence":-1} after 10000ms
    at /app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/uart.ts:299:23
    at Queue.executeNext (/app/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32)

**Comment by bjackerman** [comment]
Running...
Home Assistant 2023.5.3
Supervisor 2023.04.1
Operating System 10.1

On a Proxmox VM , passing through SONOFF_Zigbee_3.0_USB_Dongle_Plus_V2. Here is my config:

data_path: /config/zigbee2mqtt
socat:
  enabled: false
  master: pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777
  slave: tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
  options: "-d -d"
  log: false
mqtt:
  server: "!mqtt-secrets.yaml mqtt_server"
  user: "!mqtt-secrets.yaml mqtt_user"
  password: "!mqtt-secrets.yaml mqtt_password"
  base_topic: zigbee2mqtt
serial:
  port: /dev/ttyACM0
  adapter: ezsp

... and the logs ...

[06:07:15] INFO: Preparing to start...
[06:07:16] INFO: Socat not enabled
[06:07:18] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:info  2023-05-19 06:07:25: Logging to console and directory: '/config/zigbee2mqtt/log/2023-05-19.06-07-25' filename: log.txt
Zigbee2MQTT:info  2023-05-19 06:07:25: Starting Zigbee2MQTT version 1.30.4 (commit #unknown)
Zigbee2MQTT:info  2023-05-19 06:07:25: Starting zigbee-herdsman (0.14.111)
Error: Reset error: Error: {"sequence":-1} after 10000ms
    at /app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/uart.ts:299:23
    at Queue.executeNext (/app/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32)

**Comment by tomasz-kuzemko** [comment]
I had the same error happen to me today after a week of running Z2M 1.30.3 without any issues. First all the devices stopped to work and after a few minutes Z2M crashed with:

```
Error: Reset error: Error: {"sequence":-1} after 10000ms
    at /app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/uart.ts:299:23
    at runNextTicks (node:internal/process/task_queues:60:5)
    at listOnTimeout (node:internal/timers:538:9)
    at processTimers (node:internal/timers:512:7)
    at Queue.executeNext (/app/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32)
```

I have it running on k3s on a Rpi 4 and Sonoff ZBDongle-E (selected `ezsp` adater). After k3s restarted the container everything seems to be fine again. I didn't notice anything unusual in the Rpi4 system logs.

**Comment by hardwareadictos** [comment]
same here: https://github.com/Koenkk/zigbee2mqtt/issues/17412

**Comment by dazza943** [comment]
Did anyone solve this ??

**Reply by sstolk90** [reply]
Yes I did. I had tried to update a ESPHome device via HA webpage but selected by accident the Sonoff USB Zigbee stick. Same error as stated above. I shutdowned the whole system on proxmox. Removed the Sonoff USB and plugged it back in, rebooted the HA VM and it was working again.

**Reply by mdczaplicki** [reply]
@sstolk90 , jeez, I did exactly the same 😂 Thanks!

**Comment by gdschut** [comment]
I downgraded to 1.29.2 to solve this.

**Reply by Cage82** [reply]
I am having this same issue ever since upgrading to 1.32.2-1. Can you advice how you downgraded Z2M? Thanks

**Reply by gdschut** [reply]
Yes, no problem; I run Z2M in docker using docker-compose. In stead of pulling docker image "latest" i choose the "1.29.2"
was:
`image: koenkk/zigbee2mqtt:latest`
now:
`image: koenkk/zigbee2mqtt:1.29.2`
I dont know how to do it if Z2M is run any other way (I recommend docker anyway ;-) )

**Reply by Cage82** [reply]
ah ok I run it on vmware so don't have any clue about any docker =) tried to google this as well no help

**Comment by gdschut** [comment]
Now also version 1.29.2 crashes :(

**Reply by rgubin** [reply]
Having same issue with 1.32.2 run with docker
Downgrading to 1.29.2 does not helped
I enabled debugging (`-e DEBUG=zigbee-herdsman* \`) as it was suggested [here](https://github.com/Koenkk/zigbee2mqtt/issues/17412#issuecomment-1603772438) and got rid of USB hub (reconnect dongle directly to board)

**Reply by rgubin** [reply]
Ok, I turned debugging off and container start crashing again. 
[UPD] Now, after debugging enabled back, it crashes again, so that's not solution

**Comment by houseio** [comment]
same problem here, running  Home Assistant 2023.8.2 Supervisor 2023.08.1 Operating System 10.4 Frontend 20230802.0 - latest , Zigbee2MQTT Current version: 1.32.2-1


08162023_LOG:

_[13:37:13] INFO: Preparing to start...
[13:37:13] INFO: Socat not enabled
[13:37:13] INFO: Starting Zigbee2MQTT...
Error: Reset error: Error: {"sequence":-1} after 10000ms
    at /app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/uart.ts:299:23
    at Queue.executeNext (/app/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32)_

**Reply by Cage82** [reply]
I had this same. What finally helped was re-flashing the Sonoff e-dongle.

**Comment by gdschut** [comment]
What did you flash it with? Official firmware from https://github.com/itead/Sonoff_Zigbee_Dongle_Firmware/tree/master/Dongle-E/NCP?

**Reply by Cage82** [reply]
I went according to this video: https://youtu.be/KBAGWBWBATg. 

My z2m has now been running 24hrs without any crashes. Unfortunately I had already lost all my Zigbee devices due to trying million different things in random. But the difference between now and yesterday is huge. It used to crash many times an hour before this.

**Reply by Cage82** [reply]
Now running 48hrs solid, all zigbee devices still working. Also I got rid of "out of memory (node)" prompt in haos, which I thought was a separate issue from this z2m problem

**Reply by Cage82** [reply]
Well I'm back to the drawing board. Once I started adding the rest of my devices, the crashing started again. Now crashing multiple times a day like before. I'm starting to suspect my Sonoff Zigbee 3.0 USB Dongle Plus V2 model "ZBDongle-E" is the issue, starting to struggle when more and more devices are paired. This chip is marked 'experimental' here: https://www.zigbee2mqtt.io/guide/adapters/.

So now I'm looking to order a stick based on CC2652/CC1352 chip.

**Comment by gdschut** [comment]
Interesting! I have a slightly different stick: Zigbee 3.0 USB Dongle Plus–ZBDongle-E, which is based on EFR32MG21 chip: https://itead.cc/product/zigbee-3-0-usb-dongle/

**Reply by r061n** [reply]
Got it solved?
I'm having the same issue and same stick.

**Reply by gdschut** [reply]
Solved it by simply buying a new stick: Sonoff Dongle-P : https://sonoff.tech/product/gateway-and-sensors/sonoff-zigbee-3-0-usb-dongle-plus-p/
Had to reconnect all devices, which went surprisingly easy compared to the E version. Network is stable since (about 2 weeks now)

**Reply by r061n** [reply]
Lol, that's the easiest way :P
I was hoping for a solution for the Dongle-E

**Reply by reimargithub** [reply]
Same probleme here, with both sticks P and E. gdshut: Did you flashed the stick before you used it with HA ?

**Reply by gdschut** [reply]
No, I did not flash it. Just used it as I received it.

**Reply by r061n** [reply]
I bought the Dongle-P and removed: adapter: ezsp
Now it works fine!

Maybe I'll use the Dongle-E as an extender one day

**Reply by reimargithub** [reply]
Too bad...that didn't worked for my. I have both version of the dongle. Even deleting the adapter: ezsp made no difference.
I still got: 
Zigbee2MQTT:info  2023-10-12 10:49:55: Starting zigbee-herdsman (0.19.0)
Error: Reset error: Error: {"sequence":-1} after 10000ms
    at /app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/uart.ts:299:23
    at Queue.executeNext (/app/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32)

**Reply by reimargithub** [reply]
It's working again. I completly deleted the database of Zigbee2MQTT in Home Assitant, re-installed and set it up again. Now it works fine (hopefully) ;) Thanks again for the support.

**Comment by gdschut** [comment]
I had debug turned on and a crash happened. This is my log: https://pastebin.com/ux3W9Xn3

**Comment by buonhobo** [comment]
In my case the issue was that I thought I had a ZB-Dongle-E but I actually had a ZB-Dongle-P.

So I lost many hours on it and eventually fixed it by removing the `adapter: ezsp` line from the `configuration.yaml` file

**Comment by lewicki-pk** [comment]
Check if you didn't flash the **Router** firmware instead of **Coordinator**. This was a problem for me and I got the same log output.

I had the same problem, i.e. `Error: Reset error: Error: {"sequence":-1} after 10000ms (..)`. Beginners mistake:
Instead of using the Dongle-E in Coordinator mode (as it came to me out of the box) I read some instruction and flashed the device with the router software...
After flashing the NCP firmware again - everything worked fine.

**Comment by oywino** [comment]
I find it remarkable that Z2M seems to be so unreliable (meaning: To crash seems to be the solution of choice, by design?)
I've had deConz running for years, with lots of problems, but I cannot recall even once that deConz crashed. Yes, it complins and fills the log with error messages whenever something is wrong, but at least it remains standing on its two feet, and I never had to restart it to get it to resume full operation automatically once the issue has been found and fixed (device errors, network errors, coordinator errors etc.).

So why did I even consider Z2M? Well, because it is said to have much better (wider) device support, which I guess is true. But what good is that when it turns out to be so unstable and unreliable.  I noticed that every time I rebooted the coordinator, Z2M crashed. Period. 
That's not acceptable behavior!
Then one day I had to add `version: 5` to my _configuration.yaml_ because a new device needed retention set, and as a result, Z2M now crashes randomly at 5-10 hours intervals. There's very little info of any value to find in its log.
That's not acceptable behavior!

**Comment by rgubin** [comment]
Finally, firmware upgrade fixed issue.
I have Sonoff ZBDongle-E EFR32MG21, after a few hours it starts crashing with `Error: Reset error: Error: {"sequence":-1} after 10000ms` error
After upgrading from 6.x.x.x to 7.3.1.0 build 176 (latest build) using Chrome-based web updater: https://darkxst.github.io/silabs-firmware-builder/ everything worked again.

**Comment by changyeamoon** [comment]
I ended up trying all the `adapter` options until it worked! I used `adapter: zstack` to get it working!

All options in the documentation - https://www.zigbee2mqtt.io/guide/configuration/adapter-settings.html#configuration-of-the-zigbee-adapter

**Reply by TonySeeley** [reply]
This worked for me to. THANK YOU.
